src/flag-definitions.h

```diff
@@ -177,7 +177,7 @@ DEFINE_IMPLICATION(use_strong, strong_mode)
 
 DEFINE_BOOL(es_staging, false, "enable all completed harmony features")
 DEFINE_BOOL(harmony, false, "enable all completed harmony features")
-DEFINE_BOOL(harmony_shipping, true, "enable all shipped harmony fetaures")
+DEFINE_BOOL(harmony_shipping, true, "enable all shipped harmony features")
```

Comment:
Hey, this typo is still present in the codebase, why not submit a CL for this? =)